include/sdlmain.h

```diff
@@ -71,7 +69,9 @@ enum class HOST_RATE_MODE {
 	CUSTOM,
 };
 
-enum class SCALING_MODE { NONE, NEAREST, PERFECT };
+enum class SCALING_MODE { NONE, NEAREST };
+
+enum class INTEGER_SCALE_MODE { OFF, HORIZONTAL, VERTICAL };
```

Comment:
UpperCamelCase please for new enum names and values.

Reply:
IntegerScalingMode (not Scale)

Reply:
Are these styling nuances cemented anywhere? Maybe I missed some doc.

Reply:
Not sure, maybe we should start a doco. But we went over this a number of times in other PRs already in the past 1-2 years. Maybe you check out other PRs a bit more often 😄

Reply:
@johnnovak @kcgen @dreamer Perhaps it's time to update [CONTRIBUTING.md](https://github.com/dosbox-staging/dosbox-staging/blob/main/CONTRIBUTING.md)? It does establish the use of `snake_case` and `CamelCase` for class and method names , but not `UpperCamelCase` for enums.